### PR TITLE
Release :gem: 3.8.1

### PR DESCRIPTION
**Pull request by pathawks**
### Do not merge with @jekyllbot 

- [ ] Remember to squash commit before merge (and GPG sign because why not)

I _think_ the thing to do here is merge this into `3.8-stable`, and then create a merge commit to merge this branch into `master` which will make sure that `History.markdown` is the same in both branches and publish the release post on the site.

### CODE REVIEW

**Comment by pathawks** [comment]
screen shot 2018-05-01 at 11 26 55 am

File: docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: 'Jekyll 3.8.1 Released'
+date: 2018-05-01 16:00:00 +0600
+author: pathawks
+version: 3.8.1
+categories: [release]
+---
+
+Happy May Day :tada:
+
+The Jekyll team is today happy to announce the release of `v3.8.1`, which fixes
```
Comment (ashmaroli) [inline docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown]: `..is today happy..`?

File: docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+a couple of bugs that were introduced two weeks ago in `v3.8.0`. If you have
+experienced trouble regarding post excerpts or non-published posts, this release
+should be the remedy. Thanks to @Chaosed0 and @domLocalHeroes for originally
+reporting these issues, and to @ashmaroli for fixing them so quickly.
+
+As a reminder, we are now starting work on Jekyll 4.0. If there are any
```
Comment (ashmaroli) [inline docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown]: how about `we have started work on Jekyll 4.0.` ?

File: docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown (hunk 1 above)
Comment (pathawks) [reply docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown]: "...is happy to today announce..." ?

Maybe just drop "today?"

Comment (ashmaroli) [reply docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown]: yeah.. lets just drop "today" :smiley:

**Pull request merged**